process/block/headerValidator.go

```diff
@@ -53,15 +53,15 @@ func (h *headerValidator) IsHeaderConstructionValid(currHeader, prevHeader data.
 	}
 
 	if prevHeader.GetRound() >= currHeader.GetRound() {
-		log.Trace("round does not match",
+		log.Error("round does not match",
```

Comment:
Error ? this should be trace I think.

Reply:
This log is extremely useful to be here with all information as in error. It's pretty serious if we receive either a sovereign header/ headerv2/ main chain header with wrong construction.

I can change it back tough